src/components/pages/startpage/ButtonStart.js

```diff
@@ -1,84 +1,36 @@
-import React, { Component } from "react";
-import {Link } from "react-router-dom";
-
-
-
-class ButtonStart extends Component {
-    constructor() {
-        super();
-        this.state = {
-            clicked: false,
-        }
+import React from "react";
+// import {Link } from "react-router-dom";
+
+function ButtonStart(props) {
```

Comment:
Je kan je props destructuren zoals dit ButtonStart({name}) zo kan je duidelijk zien welke props je krijgt en hoef je niet overal props. voor te zetten. is vooral handig als je wat meer props hebt